app/frontend/wwwroot/service-worker.published.js

```diff
@@ -37,7 +37,12 @@ async function onFetch(event) {
     if (event.request.method === 'GET') {
         // For all navigation requests, try to serve index.html from cache
         // If you need some URLs to be server-rendered, edit the following check to exclude those URLs
-        const shouldServeIndexHtml = event.request.mode === 'navigate';
+        const shouldServeIndexHtml = event.request.mode === 'navigate'
```

Comment:
What's this for? It assumes that they've installed the app as a PWA.

Reply:
It's to enable `api` and `.auth`  for server-render. Otherwise those apis will just redirect to `index.html`